modules/backend/tests/classes/NavigationManagerTest.php

```diff
@@ -26,14 +26,14 @@ public function testRegisterMenuItems()
         $this->assertArrayHasKey('WINTER.TEST.DASHBOARD', $items);
 
         $item = $items['WINTER.TEST.DASHBOARD'];
-        $this->assertObjectHasAttribute('code', $item);
-        $this->assertObjectHasAttribute('label', $item);
-        $this->assertObjectHasAttribute('icon', $item);
-        $this->assertObjectHasAttribute('url', $item);
-        $this->assertObjectHasAttribute('owner', $item);
-        $this->assertObjectHasAttribute('order', $item);
-        $this->assertObjectHasAttribute('permissions', $item);
-        $this->assertObjectHasAttribute('sideMenu', $item);
+        $this->assertObjectHasProperty('code', $item);
```

Comment:
@wverhoogt with PHPUnit changes like this where it's a simple name change, I like to create stubs in the root `TestCase` class ([see here for example](https://github.com/wintercms/winter/blob/develop/modules/system/tests/bootstrap/TestCase.php#L61-L76)) so that the old names continue to work.

I'm happy for us to keep the changed names here, I'd just still like the old method name to still work so people don't have to update their tests.

Reply:
I´ve added the stub method.